app/frontend/src/store/modules/form.js

```diff
@@ -18,7 +18,7 @@ const genInitialForm = () => ({
   snake: '',
   submissionReceivedEmails: [],
   userType: IdentityMode.TEAM,
-  versions: [],
+  versions: []
```

Comment:
you can get the form versions from the form itself:

https://github.com/bcgov/common-hosted-form-service/compare/master...jasonchung1871:common-hosted-form-service:export-versioning